### PR TITLE
Updates devtest to have some requested stuff.

### PR DESCRIPTION
**Pull request by Laboredih123**
[MAPPING][QOL]
## About the PR 
Changes area typepath to allow random events.
Fixes turbine connections.
Expand mechanic zone and make it easier to move around in, also connects it to the mainframe.
Fix icons in pumps.
Rearrange gas canisters to be more accessible.
Rearrange HE pipes to be more efficient.
Moves cloning and adds genetics.
Rearrange Botany to be more accessible, also removes some redundancy in it.
Merge Robotics and medical.
Makes the test chamber more useful with toys and wires for testing.
Rearrange Toxins to be more movable in and useful at startup.
Moves mining to be more accessible.
Minor quality of life changes such as chairs and doors to the singulo.

## Why's this needed? 
Having a nice and useful developer testing map is what devtest is for.

### CODE REVIEW

**Comment by DasBrain** [comment]
Can I get a law rack too?

**Comment by Laboredih123** [comment]
i already put one in

**Comment by ZeWaka** [comment]
Draft.

**Comment by Laboredih123** [comment]
yes

**Comment by Laboredih123** [comment]
:(

**Comment by ZeWaka** [comment]
this PR either needs to have no more additions or split up for atomization concerns

File: code/area.dm [hunk 1]
```diff
@@ -4138,8 +4138,9 @@ ABSTRACT_TYPE(/area/mining)
 	teleport_blocked = 1
 	icon_state = "purple"
 
-/area/devzone
-	name = "Dev Area"
+/// For Devtest testing purposes
+/area/station/devzone
+	name = "Dev Zone: Mostly Harmless"
```
Comment (pali6) [inline code/area.dm:4143]:
```suggestion
	name = "Dev Zone"
```

Comment (Laboredih123) [reply code/area.dm:4144]: alas, no fun. understood

**Pull request merged**